libs/sdk-bindings/bindings-android/lib/src/main/kotlin/breez_sdk_notification/ServiceLogger.kt

```diff
@@ -0,0 +1,20 @@
+package breez_sdk_notification
+
+import android.util.Log
+import breez_sdk.LogEntry
+import breez_sdk.LogStream
+
+class ServiceLogger(private val logger: LogStream?) {
```

Comment:
Why do we need ServiceLogger and not pass directly LogStream? The tag can be constant for logs coming from the extension.